.github/workflows/deploy.yml

```diff
@@ -49,6 +49,8 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Update Postman Collection
+        env :
+          POSTMAN_API_KEY: ${{ secrets.POSTMAN_API_KEY }}
         run: |
           chmod +x -R ./.github/
-          ./.github/scripts/postmanUpdate.sh PMAK-66221c18ec460900018fc595-1e8e3950bba1f17e6ee46fd5e971c6d208
+          ./.github/scripts/postmanUpdate.sh ${{ secrets.POSTMAN_API_KEY }}
```

Comment:
J'ai repris la clés:
PMAK-66221c18ec460900018fc595-1e8e3950bba1f17e6ee46fd5e971c6d208
que j'ai foutu dans le secret, comme pour le github Actions, si ça te va tu peux merge ! 

![image](https://github.com/MathysHaubert/Event-it-API/assets/46419378/45fd99b3-631a-4ff5-9b03-d91f435264a3)